docs/concepts/vega-chain.md

```diff
@@ -0,0 +1,127 @@
+
+When a validator's token limit is reached, and more nomination would cause a validator's maximum stake to be exceeded, any additional nominated tokens will not be used. The remaining amount will be available to use to nominate another validator after the next epoch has begun.
+
+**Un-nominating validators** 
+Participants can remove their nomination at the end of an epoch, or immediately. The un-nominated tokens will be restored back to the participant's associated token balance.
+
+_Un-nominate towards the end of the epoch_ 
+A participant can un-nominate towards the end of the current epoch, which means the stake is not used for the validator from the following epoch. The action is announced in the next available block of the same epoch, but the nominator keeps their nomination active until the last block of that epoch. At that point, the tokens are returned. The nominator cannot move those tokens before the epoch ends.
+
+_Un-nominate Now_ 
+A participant can choose to un-nominate at any time, and the action is executed immediately following the block it is announced in (within the same epoch). The participant will not receive any rewards from the validator in that epoch. The tokens are marked as available to the participant.
+
+#### Automatic delegation (WIP)
+- A party becomes eligible to participate in auto delegation once they have manually nominated 95%+ of the association.
+- Once entering auto delegation mode, any un-nominated associated tokens will be automatically distributed according to the current validator nomination of the party maintaining the same proportion.
+
+#### Rewards (restricted mainnet)
+Validators and nominators both receive incentives from the network, depending on factors including how much stake is nominated. 
+
+For restricted mainnet, all rewards are evenly distributed among validators. 
+
+At the end of each epoch, reward payments are calculated per active validator, and then some of that reward is divided between their nominators. The proportion that goes to nominators is defined by the network parameter `reward.staking.delegation.delegatorShare`. 
+ 
+:::info
+VEGA tokenholders can use [token.vega.xyz](https://token.vega.xyz) to associate their tokens and nominate validators to receive rewards. CoinList
+custodial users should confirm with CoinList how staking works for them.
+:::
+
+### ***Further reading*** 
+Read the [staking rewards](https://github.com/vegaprotocol/specs/blob/main/protocol/0058-simple-POS-rewards.md) spec for full details for how rewards are calculated. 
+
+#### Bridges used for staking
+Because VEGA is an ERC20 token, all actions regarding staking are initiated on Ethereum, rather than on the Vega protocol. This is allows VEGA to be staked to a Vega public key without any action on the Vega network, and without putting the tokens under the control of the Vega network.
```

Comment:
```suggestion
Because VEGA is an ERC20 token, all actions regarding staking are initiated on Ethereum, rather than on the Vega protocol. This allows VEGA to be staked to a Vega public key without any action on the Vega network, and without putting the tokens under the control of the Vega network.
```